Homework 5/Task 5.1/SparseMatrix.cs

```diff
@@ -0,0 +1,187 @@
+﻿using System.Collections;
+using System.Text;
+
+namespace Homework_5.Task_5._1
+{
```

Comment:
Maybe it works but too complicated and unclear logics. Please rework the task using standard collection , Dictionary<(int,int), long > for example. In current implementation it is absolutely not clear what is CornerElements